resotoworker/resotoworker/collect.py

```diff
@@ -68,6 +69,10 @@ def collect(collectors: List[BaseCollectorPlugin]) -> Graph:
 
         collected = collect(collectors)
 
+        for post_collector_class in post_collectors:
+            instance = post_collector_class()
```

Comment:
This and the next line should be wrapped in a `try/except Exception` block. Currently an exception in a post collector will abort the entire collect run.